pykube/objects.py

```diff
@@ -281,6 +281,14 @@ def unschedulable(self):
             return self.obj["spec"]["unschedulable"]
         return False
 
+    @property
+    def ip(self):
+        addresses = self.obj["status"]["addresses"]
+        for address_data in addresses:
+            if address_data["type"] == "InternalIP":
```

Comment:
Is type always present? This will throw an exception if any of these keys don't exist. Should that be caught? Or just check for existence?